compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -145,7 +145,7 @@ public class BIRPackageSymbolEnter {
     private BStructureTypeSymbol currentStructure = null;
     private LinkedList<Object> compositeStack = new LinkedList<>();
 
-    private static final int SERVICE_TYPE_TAG = 53;
+    private static final int SERVICE_TYPE_TAG = 54;
```

Comment:
Why do we have to change this?

Reply:
We are using this just as the limit to throw an `AssertionError()` https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java#L1651. 
We added regexp tag to TypeTags; hence the limit went up.